src/input/field/field.jsx

```diff
@@ -4,7 +4,9 @@ import { onlyUpdateForPropTypes } from 'recompose'
 import classnames from 'classnames'
 import styles from '../../../src/input/input.styles.scss'
 
-const Field = ({ input, placeholder, type, meta: { touched, error, submitFailed }, ...rest }) => {
+const Field = ({
```

Comment:
I think the fix is more about removing `onlyUpdateForPropTypes` since we use `...rest`.
Can you try like that ?

Reply:
You should let the `input` fix as it is.
But removing `onlyUpdateForPropTypes` just here

Reply:
I already try this, but it doesn't work

Reply:
Did you try to 
 - let Input fix you did (declare onPaste on **input**)
 - remove the `onlyUpdateForPropTypes` on **field**

(both modifications ?)

Reply:
Sorry, I didn't understand :) this solution works